wagtail/admin/static_src/wagtailadmin/js/vendor/bootstrap-tab.js

```diff
@@ -1,138 +0,0 @@
-/* ========================================================================
- * Bootstrap: tab.js v3.0.0
- * http://twbs.github.com/bootstrap/javascript.html#tabs
- * ========================================================================
- * Copyright 2012 Twitter, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- * ======================================================================== */
-
-
-+function ($) { "use strict";
```

Comment:
I’m so glad we can remove all of this! @harrislapiroff did you spend much time looking at this implementation to make sure the new one matches, or was that not worth it?

Reply:
I didn't spend a lot of time checking it bit for bit, but the tabs _seem_ to be working the same? 😬

Reply:
😬 great!